### ISSUE TITLE
Question: Documentation

### ISSUE THREAD

**Issue by DerZwergGimli** [openviess/PyViCare#227]
Hi,

I would like to know if there is an documentation about this project.

More specifically i would like to know if there is a list or doc about all the get/set functions for the different devices like: burner/circuit/...

My goal would be to loop through all getter functions of 'burner' for exmaple.

Currently i'm using the tests examples to figure out whats the functions are for the topics.
There has do be a better way?!

Thanks

**Comment by DerZwergGimli** [comment]
...edit: Is this the way to go?

`methodList = [method for method in dir(PyViCareGazBoiler.GazBoiler) if method.startswith('get') is True]`
`print(methodList)`

**Comment by woehrl01** [comment]
No, currently there is no additional documentation than the [readme](https://github.com/somm15/PyViCare/blob/master/README.md). 

There are examples. Especially as for the use case you describe, here: https://github.com/somm15/PyViCare/blob/master/tests/test_Integration.py

If you think a more detailed documentation is useful, feel free to contribute.

**Comment by DerZwergGimli** [comment]
> If you think a more detailed documentation is useful, feel free to contribute.

Yeah maybe i will do this if i can figure it all out :D

**Comment by DerZwergGimli** [comment]
What does 'Gaz' stand for? Is it equal to "Gas"?

**Comment by woehrl01** [comment]
Yes, it's French for "gas". Changing this would be a breaking change for our users, which is currently not worth the trouble.
